src/tests/tests.yaml

```diff
@@ -1,9 +1,17 @@
 charm_name: cinder-netapp
 tests:
-  - tests.tests_cinder_netapp.CindernetappTest
+  - zaza.openstack.charm_tests.cinder_netapp.tests.CinderNetAppTest
 configure:
   - zaza.openstack.charm_tests.keystone.setup.add_demo_user
 gate_bundles:
-  - xenial-stein
+  - bionic-queens
+  - bionic-rocky
+  - bionic-stein
+  - bionic-train
+  - bionic-ussuri
+  - focal-ussuri
+  - focal-victoria
+  - groovy-victoria
+  - hirsute-wallaby
```

Comment:
Note that [we are in the process of adding also focal-xena and impish-xena to all charms](https://review.opendev.org/q/topic:xena-bundles) so you probably want to do the same for this charm as well